sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs

```diff
@@ -30,8 +31,6 @@ public partial class EventHubOptions : Microsoft.Azure.WebJobs.Hosting.IOptionsF
         public bool InvokeProcessorAfterReceiveTimeout { get { throw null; } set { } }
         public string LeaseContainerName { get { throw null; } set { } }
         public int MaxBatchSize { get { throw null; } set { } }
-        public void AddEventHubProducerClient(Azure.Messaging.EventHubs.Producer.EventHubProducerClient client) { }
```

Comment:
I removed these members for now as we don't have a consistent story for the processor. I'd like to get some preview feedback before adding them back.